components/nifosg/controller.cpp

```diff
@@ -85,7 +85,7 @@ KeyframeController::KeyframeController(const KeyframeController &copy, const osg
 }
 
 KeyframeController::KeyframeController(const Nif::NiKeyframeData *data)
-    : mRotations(data->mRotations)
+    : mRotations(data->mRotations, osg::Quat())
```

Comment:
Where is the bug? The second argument is optional and will be value-initialized to osg::Quat() if it's missing. Specifying the argument explicitely doesn't hurt of course.